tests/make-nohistory-image

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# make-no-history-image - craft a custom image used in bud-implicit-no-history test.
+#
+# This script does not run in CI, or automated tests, or gating. In fact
+# it is meant to be run once, only once, by a human, and then (with luck)
+# never again. The purpose is to create a custom test image with certain
+# very specific attributes. This image will be used in one specific test.
+# Maybe some day other tests. The image should be static and immutable.
+# This creation script is checked into the buildah source tree simply
+# as reference for how the image was created, and as a tool should there
+# ever be a need to update that test image in some way.
+#
+# See https://github.com/containers/buildah/pull/5473
+#
+set -e
+
+# Name of the resulting image. (Never pushed automatically).
+# If image build is successful, a human can push it to quay.
+IMAGE=quay.io/libpod/buildah-testimage-nohistory:$(date +%Y%m%d)
+
+function createrandom() {
+    outfile=${1?Missing OUTFILE arg to createrandom()}
+    if [[ ! -e $outfile ]]; then
+        dd if=/dev/urandom bs=1 count=${2:-256} of=$outfile status=none
+    fi
+}
+
+workdir=$(mktemp -d)
+
+# First layer is a copy of this script
+# FIXME: find some way to add a label or annotation or something to indicate that
+cp $0 $workdir/file01
+cd $workdir
+
+mkdir -p oci/blobs/sha256
+
+# Build an image config and image manifest in parallel
+# FIXME: find some way to create images for other arches?
+echo '{"architecture": "amd64", "os": "linux", "rootfs": {"type": "layers", "diff_ids": [' > config.json
```

Comment:
To add labels, add `"labels": {"key1": "value1" [, "key2": "value2" [...]]},` between `"linux",` and `"rootfs":`.